Packs/NetscoutAED/Integrations/NetscoutAED/NetscoutAED.yml

```diff
@@ -0,0 +1,1286 @@
+- display: Server URL
+  name: base_url
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- name: User
+  displaypassword: API Key
+  required: false
+  hiddenusername: true
+  type: 9
+description: Use the Netscout Arbor Edge Defense Integration to detect and stop both
+  inbound threats and outbound communication from internal compromised hosts.
+display: Netscout Arbor Edge Defense
+name: NetscoutAED
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: An ISO-standardized country code to get a specific country in the
+        results. Can be retrieved by running the "na-aed-country-code-list" command.
+      isArray: false
+      name: country
+      required: false
+      secret: false
+    - default: false
+      description: 'Search strings, separated by “+” to filter the results. (example:
+        "AZ+BS").'
+      isArray: false
+      name: query
+      required: false
+      secret: false
+    - default: false
+      description: The page of the results to return.
+      isArray: false
+      name: page
+      required: false
+      secret: false
+    - default: false
+      description: Maximal number of results to retrieve. Also sets the size of the
+        returned page.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    deprecated: false
+    description: Gets the countries on the outbound blacklist. By default, 10 blacklisted
+      countries are returned.
+    execution: false
+    name: na-aed-outbound-blacklisted-countries-list
+    outputs:
+    - contextPath: NetscoutAED.OutboundBlacklistCountry.annotation
+      description: A message associated with each country in the outbound blacklist.
+      type: String
+    - contextPath: NetscoutAED.OutboundBlacklistCountry.country
+      description: An ISO standardized country code.
+      type: String
+    - contextPath: NetscoutAED.OutboundBlacklistCountry.update_time
+      description: The time that the country code was added to the list.
+      type: Date
+  - arguments:
+    - default: false
+      description: 'Search strings, separated by “+” to filter the results. (example:
+        "AZ+BS")'
+      isArray: false
+      name: query
+      required: false
+      secret: false
+    - default: false
+      description: The page of the results to return.
+      isArray: false
+      name: page
+      required: false
+      secret: false
+    - default: false
+      description: Maximal number of results to retrieve. Also sets the size of the
+        returned page (default 10)
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    deprecated: false
+    description: Gets a country or list of countries (country name and ISO-standardized
+      country code).
+    execution: false
+    name: na-aed-country-code-list
```

Comment:
Please verify the name of the commands with arseny (na-aed)

Reply:
Verified